### PR TITLE
Initial karma test and setup

### PR DESCRIPTION
**Pull request by AndreWillomitzer**
Hi there, good idea about the PR now we can see the conflicts right in front of us and hopefully resolve it. Basically the only things we care about are the changes to package.json, karma.conf.ts, and the karma test.

### CODE REVIEW

File: karmaTests/simpleKarma.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const assert = require('chai').assert
+it('should equal 2', () => {
```
Comment (kibertoad) [inline karmaTests/simpleKarma.ts:2]: can you wrap it in describe block as well?

File: karma.conf.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+module.exports = function (config: any) {
```
Comment (kibertoad) [inline karma.conf.ts:1]: you barely use any ts features here, does it need to be ts and not js?

Comment (AndreWillomitzer) [reply karma.conf.ts:1]: ehhh probably not. I guess I just wanted to be uniform with the project lol

File: karmaTests/simpleKarma.ts (hunk 1 above)
Comment (AndreWillomitzer) [reply karmaTests/simpleKarma.ts:2]: The initial test I used from Karma site did have a describe but then it basically said the same 0 of 0 success so I made a more simple test like this one but same result :(

File: karma.conf.ts (hunk 2 above)
Comment (AndreWillomitzer) [reply karma.conf.ts]: Hi sir, I tried to update to JS and basically had to go back to commonJS "export default function" style. But when I do, and try to run the tests it says:

image

And if I try to use require it's not defined. I try to use module.exports and it complains about needing to use export default.

Any ideas?

Comment (AndreWillomitzer) [reply karma.conf.ts]: @kibertoad So it's basically telling me/forcing me to use export default, and when I do it says no you can't do that...

**Pull request closed without merging**